frontend/src/components/SectionNavs/SectionNavs.js

```diff
@@ -0,0 +1,64 @@
+import Box from '@mui/material/Box'
+import Link from '@mui/material/Link'
+import { useCallback, useState } from 'react'
+import { useLocation } from 'react-router-dom'
+
+import { slugify } from 'utils/helpers'
+import useScrollSpy from 'utils/scrollSpy'
+
+export const SectionNavs = ({ sections }) => {
+  const location = useLocation()
+  const [locationHash, setLocationHash] = useState(location.hash)
+
+  const handleScroll = useCallback(element => {
+    const hash = `#${element.id}`
+    window.history.pushState(null, null, hash)
+    setLocationHash(hash)
+  }, [])
+
+  const handleNav = useCallback(event => {
+    event.preventDefault()
+    const hash = event.target.href.split('#')[1]
+    const elem = document.getElementById(hash)
+    elem.scrollIntoView({ behavior: 'smooth' })
+    setLocationHash(`#${hash}`)
+  }, [])
+
+  useScrollSpy({ onScroll: handleScroll })
+
+  return (
+    <Box sx={{ position: 'fixed', width: '100%', zIndex: 'appBar' }}>
```

Comment:
Should be something like `position: 'sticky', top: theme.scrolledHeaderHeight`.

Reply:
We can't just give the scrolledHeaderHeight, because this sticky navbar is going to be used in other pages with different header height. We should use a kind of React context to get the header height of the current page.

Reply:
Works for me. 👍